_posts/2020-04-09-status-update.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "8th April 2020: IAC Team Status Update"
+author: michaeltlombardi
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - powershell
+  - windows
+---
+
+The IAC team has been hard at work over the past week.
+Here's a brief overview of what we've been up to:
+
+## New Module Releases (IAC Team)
+
+Let's kick things off by highlighting a new module release:
+
+- [puppetlabs/acl](https://forge.puppet.com/puppetlabs/acl) (v3.1.1)
+
+## Community Contributions
+
+As always, we work hand-in-hand with the broader Puppet community to get work done.
+This week, we want to take a moment to highlight a few PRs and the people behind them!
+
+First up, a [pull request to the puppetlabs/docker module][community-seriv-pr] by [Sergey Ivanov][community-seriv].
+Sergey's work enabled the module to function transparently on RedHat systems utilizing the [`podman-docker` package!][podman-docker].
+Thank you, Sergey!
+
+We'd like to thank [Atanas Stoyanov][community-amorphina] for [his fix to the Apache module][community-amorphina-pr].
+This PR ensures that the custom ldap options in `/etc/apache2/mods` can actually be applied and loaded, fixing a previously unknown bug!
+Thank you, Atanas!
+
+Finally, big Thanks to [Frank Wall][community-fraenki] for [his PR][community-fraenki-pr] that will fix [MODULES-10023][modules-10023], ensuring that the xtrabackup functionality in [puppetlabs/mysql][puppetlabs/mysql] actually works as intended!
+Thank you, Frank!
+
+## Supported Module Litmus Conversion
+
+On the topic of [Litmus][litmus-gem-home], we're continuing to make progress on the conversion of all supported modules to the framework:
+
+- [puppetlabs/reboot][puppetlabs/reboot] is completed, running against all 33 of the supported platforms!
+- [puppetlabs/websphere_application_server][puppetlabs/websphere_application_server] is wrapped!
+- [puppetlabs/satellite_pe_tools][puppetlabs/satellite_pe_tools] is still progressing nicely!
+- [puppetlabs/ibm_installation_manager][puppetlabs/ibm_installation_manager] is in progress now!
+
+After that, we'll be moving on to
+
+## Puppet + DSC: Phase II Begins!
+
+[michaeltlombardi][puppet-michael] has continued to make excellent progress on our [refactor of DSC support][iac-41] in Puppet as we step in [Phase II][iac-685]!
```

Comment:
Given you're the author, I'm wondering if you want to state, "I've continued to make great progress on..." or something along those lines?

Reply:
I think I'll change this out for `we` to be honest. It's a team effort.

Reply:
I still think you deserve 100% of the credit, so I wouldn't change that fact. Maybe as we contribute more in future iterations, but this has been you doing the lion's share of the work!